neuralmonkey/tests/test_nn_utils.py

```diff
@@ -34,6 +34,18 @@ def test_keep_prob(self):
             expected_dropped_size = 10000 * (1 - kprob)
 
             self.assertTrue(np.isclose(expected_dropped_size, dsize, atol=100))
+            
+    def test_train_false(self):
+        """Checks that dropout does not """
```

Comment:
sakra jsem zacal resit jak spravne udelat unit test a nedopsal jsem to :D ... tak pak to pridam k poslednimu commitu, ale mimo to by to takhle mohlo uz byt?